homeassistant/components/velbus/__init__.py

```diff
@@ -62,7 +62,11 @@ def callback():
         load_platform(hass, 'sensor', DOMAIN,
                       discovery_info['sensor'], config)
 
+    def syn_clock(self):
+        controller.sync_clock()
+
     controller.scan(callback)
+    hass.services.async_register(DOMAIN, 'sync_clock', syn_clock)
```

Comment:
Add `schema=vol.Schema({})` so validation fails if a user accidentally passes in data.